pkg/odo/cli/application/application.go

```diff
@@ -32,7 +32,7 @@ func NewCmdApplication(name, fullName string) *cobra.Command {
 		Use:   name,
 		Short: "Perform application operations",
 		Long:  `Performs application operations related to your OpenShift project.`,
-		Example: fmt.Sprintf("%s\n%s\n%s",
+		Example: fmt.Sprintf("%s\n\n%s\n\n%s",
```

Comment:
The other command help like catalog, config and also rest of the command help follows the single spacing format. Even look and feel point of view single space in string looks like an ideal format.

TBH i am not in the favor of this change

Reply:
> The other command help like [catalog](https://github.com/openshift/odo/blob/master/pkg/odo/cli/catalog/catalog.go#L26), [config](https://github.com/openshift/odo/blob/master/pkg/odo/cli/config/config.go#L30) and also rest of the command help doesn't follow two new line spacing spacing format. Even UX look and feel point of view single new line spacing looks an ideal format to me.
> 
> TBH this change doesn't make sense to me unless there is a hard requirement to do it cc_ @surajnarwade @mik-dass

Ahh... 
After testing the pr i can see the difference. I see the rest of cli command help, new line spacing is implemented in a bit different way i mean in the sub command file. 

I am fine with the current change.

Reply:
Yup. It's implemented differently. We'll have to refactor this later haha.

Reply:
Can we please have an issue to track that @cdrage 🙂